torch/_dynamo/variables/nn_module.py

```diff
@@ -303,7 +303,7 @@ def call_function(
                     # the call_wrapped currently, and maybe other issues too
                     fn = mod.forward
                 else:
-                    fn = mod.__call__
+                    fn = mod._call_impl
```

Comment:
So the inline test works but I think this ended up having a huge accuracy implication, so many tests broken cc @yanboliang

Reply:
I just checked and found this is because for ```UnspecializedNNModule```, we are still going to inline ```__call__```, I think if you update the it to ```_call_impl```, it should work well.
https://github.com/pytorch/pytorch/blob/dc4a25312f6eec5d9790d363ae52433c62543651/torch/_dynamo/variables/nn_module.py#L693